tfjs-core/src/ops/browser.ts

```diff
@@ -115,43 +115,82 @@ function fromPixels_(
   }
 
   const [width, height] = isVideo ?
-      [
-        (pixels as HTMLVideoElement).videoWidth,
-        (pixels as HTMLVideoElement).videoHeight
-      ] :
-      [pixels.width, pixels.height];
-  let vals: Uint8ClampedArray|Uint8Array;
+  [
+    (pixels as HTMLVideoElement).videoWidth,
```

Comment:
Why you need to change the original fromPixels_ function?

Reply:
Done.